src/strpak.cpp

```diff
@@ -391,6 +392,14 @@ char* strncpy0(			// copy/truncate with 0 fill
 	return d;
 }		// ::strncpy0
 // ====================================================================
+char* strt_string_view(		// copy string_view to temporary
+	std::string_view sv)	// string view
+{
+	// strncpy0 handles sv.size() = 0
+	char* sRet = strncpy0( nullptr, sv.data(), sv.size()+1);
+	return sRet;
+}		// :: strt_string_view
+// ====================================================================
```

Comment:
Is this still needed?

Reply:
Not right now, but I made it and tested it.  It is tiny and likely useful soon given the ever-growing interaction with other libraries.  So I say leave it.